test/integration/features/coolstore.feature

```diff
@@ -0,0 +1,96 @@
+@coolstore
+Feature: Cool Store
+  In order to test Minishift under load
+  I need to setup a test environment
+  Using CentOS and 4G of memory
+  After which I will import image streams
+  Finally I will deploy the Cool Store
+  
+  Scenario: Enabling 'xpaas' add-on
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift addons enable xpaas"
+     Then stdout should contain
+      """
+      Add-on 'xpaas' enabled
+      """
+
+  Scenario: Starting Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --memory 4096 --iso-url centos" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: User can create new namespace 'coolstore'
+   Given Minishift has state "Running"
+    When executing "oc new-project coolstore" succeeds
+    Then stdout should contain
+     """
+     Now using project "coolstore"
+     """
+  
+  Scenario: User can create a template for coolstore
+   Given Minishift has state "Running"
+    When executing "oc create -f ./templates/coolstore.yaml" succeeds
+    Then stdout should contain
+     """
+     template "coolstore" created
+     """
+
+  Scenario: User can deploy the coolstore template to namespace 'coolstore' 
+   Given Minishift has state "Running"
+    When executing "oc new-app --template=coolstore" succeeds
+    Then stdout should contain
+     """
+     Success
+     """
+
+  Scenario: Successfully deployed services to namespace coolstore
```

Comment:
I don't think this is a different scenario, since the deployment of the services is triggered by the _oc new-app_ above. I think it more like this:

    Scenario: User can deploy the coolstore app to namespace 'coolstore' 
      Given Minishift has state "Running"
       When executing "oc new-app --template=coolstore" succeeds
       Then stdout should contain
        """
        Success
        """
        And services "web-ui, inventory, catalog, cart, coolstore-gw" rollout successfully